colossus-metrics/src/main/scala/colossus/metrics/collectors/Histogram.scala

```diff
@@ -97,11 +97,11 @@ object Histogram extends CollectorConfigLoader{
     * @param collection The collection which will contain this Collector.
     * @return
     */
-  def apply(address : MetricAddress, configPath : String)(implicit collection : Collection) : Histogram = {
+  def apply(address : MetricAddress, configPath : String)(implicit ns : MetricNamespace) : Histogram = {
 
     import scala.collection.JavaConversions._
 
-    val params = resolveConfig(collection.config.config, s"$configPath.$address", DefaultConfigPath)
```

Comment:
config.config makes me sad.  I've bought this upon us, I'll see if i can make it better somehow